third_party/mustache/README.amp

```diff
@@ -0,0 +1,10 @@
+URL: https://github.com/janl/mustache.js/
+License: The MIT License
+License File: https://github.com/janl/mustache.js/blob/master/LICENSE
+
+Description:
+Fork of mustache.js library.
+
+Local Modifications:
+None yet, but expected changes for restricting instructions and sanitizing
```

Comment:
Just for process. We want a commit that has the original sources. Ideally you have a commit that has just the third_party addition in your PR and then a second commit with all the other changes.